cmd/vic-machine/main.go

```diff
@@ -62,9 +63,23 @@ func main() {
 			Action: inspect.Run,
 			Flags:  inspect.Flags(),
 		},
+		{
+			Name:   "version",
+			Usage:  "Show VIC version information",
+			Action: showVersion,
+		},
+	}
+	if Version != "" {
```

Comment:
We need release version, but at this time, we cannot get that value, for we don't build after the build version is set.
While we have release version, e.g. tag set, to trigger build, this version can be set, and then we can get version like <release version>-<build number>-<commit sha>